webapp/src/ts/modules/analytics/analytics-target-aggregates.component.ts

```diff
@@ -0,0 +1,81 @@
+import { Component, OnDestroy, OnInit } from '@angular/core';
+import { Store } from '@ngrx/store';
+import { combineLatest, Subscription } from 'rxjs';
+
+import { TargetAggregatesActions } from '@mm-actions/target-aggregates';
+import { TargetAggregatesService } from '@mm-services/target-aggregates.service';
+import { Selectors } from '@mm-selectors/index';
+
+@Component({
+  selector: 'analytics-target-aggregates',
+  templateUrl: './analytics-target-aggregates.component.html',
+})
+export class AnalyticsTargetAggregatesComponent implements OnInit, OnDestroy {
+  private targetAggregatesActions: TargetAggregatesActions;
+  subscriptions: Subscription = new Subscription();
+  loading = true;
+  enabled = false;
+  aggregates = null;
+  selected = null;
+  error = null;
+
+  constructor(
+    private store: Store,
+    private targetAggregatesService: TargetAggregatesService
+  ) {
+    this.targetAggregatesActions = new TargetAggregatesActions(store);
+  }
+
+  ngOnInit(): void {
+    this.subscribeToStore();
+    this.getTargetAggregates();
+  }
+
+  ngOnDestroy(): void {
+    this.subscriptions.unsubscribe();
+    this.targetAggregatesActions.setTargetAggregatesError(null);
+    this.targetAggregatesActions.setTargetAggregates(null);
+    this.targetAggregatesActions.setTargetAggregates(false);
```

Comment:
I think this was meant to be "setTargetAggregatesLoaded(false)"? :) maybe this deserves a unit test.

Reply:
😩 totally, thanks! Fixed

Reply:
Nope, the issue still there :(